index.js

```diff
@@ -58,6 +58,34 @@ function onInitHooks() {
 	)
 }
 
+function createSuperuser() {
+
+	const { SUPERUSER_EMAIL, SUPERUSER_PASSWORD } = require('./src/lib/config');
+
+	if (!SUPERUSER_EMAIL || !SUPERUSER_PASSWORD) {
+		return;
+	}
+
+	console.log("Creating superuser account...")
```

Comment:
Prettier is not processing this file because it's plain Javascript and I think we configure it to only touch .ts files in src/.